include/uapi/linux/fanotify.h

```diff
@@ -38,10 +38,11 @@
 #define FAN_UNLIMITED_MARKS	0x00000020
 #define FAN_ENABLE_AUDIT	0x00000040
 #define FAN_UNPRIVILEGED	0x00000080
-
+#define FAN_EVENT_INFO_TID	0x00000100	/* reports the thread id of the event trigger */
```

Comment:
please take care for line spacing and stick to existing code practices.
IOW, don't remove the empty line.